platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -206,6 +142,78 @@ The existing method does not behave as specified by its javadoc, and we do not p
 
 Potential migrations include using `Object.equals(Object)` directly, or comparing the fields of dependencies manually.
 
+[[changes_8.10]]
```

Comment:
There were a number of deprecations added in 8.11 that were erroneously added to the 8.10 section. They were moved up to 8.11 in this PR. That is why the diff looks like this